### ISSUE TITLE
network 1.16-378: ERROR

### ISSUE THREAD

**Issue by martinamorris** [statnet/network#19]
Started because CRAN sent an email to @CarterButts saying `network` was generating errors with a new code-checking package `rchk` _for the development version of R_, and that these errors needed to be fixed by a specific date or else...  See initial discussion in #15

Most of this exchange occurred in email, and I'm transporting it here for the record.

1. Carter was unable to reproduce the errors using our regular build tools, and there is more than one version of `rchk` around, so it was not clear which one CRAN is using.  
2. @krivit found no errors using `valgrind`  
3. @krivit and @chad-klumb  implemented different versions of `rchk`, each of which generated a long list of errors associated with `protect` stacks.

* Chad's implementation came from R-hub

![image](https://user-images.githubusercontent.com/8194530/69289348-0fc66a80-0bb1-11ea-9cd4-0c5dddcb076e.png)

Pavel's implementation:

> I rebuilt the singularity image based on the recipe in
> https://raw.githubusercontent.com/kalibera/rchk/master/image/singularity.def
> , so it's against the latest R-devel and versions of everything, as far
> as I can tell.

and he also noted:

> Regarding the protection depth, I found a ticket about that: https://github.com/kalibera/rchk/issues/21 [github.com] ,
> which suggests that the checker can't handle situations in which PROTECT() and UNPROTECT() are inside a loop.

Meanwhile, however, @CarterButts says that CRAN has removed the link to the `rchk` output file that they sent him, so it is not possible to compare what Pavel and Chad found to what CRAN is complaining about.

Bottom line for now:  We'll submit an update for `network` with the fix to is.color(), and ignore the rest.  If CRAN wants to complain about `rchk` results we'll follow up.

**Comment by martinamorris** [comment]
fixed with commit 746a5320dddaf4cf9e259a21ccf742ca488ec5df and 1.16 release
